### ISSUE TITLE
Linking error for android templates on Win10 with mingw

### ISSUE THREAD

**Issue by Abhicse079** [godotengine/godot#3265]
```
drivers\libdrivers.android.debug.armv7.neon.a(bands.android.debug.armv7.neon.o):bands.c:function compute_theta: error: undefined reference to 'stereo_itheta'
drivers\libdrivers.android.debug.armv7.neon.a(bands.android.debug.armv7.neon.o):bands.c:function compute_theta: error: undefined reference to 'isqrt32'
drivers\libdrivers.android.debug.armv7.neon.a(bands.android.debug.armv7.neon.o):bands.c:function compute_theta: error: undefined reference to 'isqrt32'
drivers\libdrivers.android.debug.armv7.neon.a(bands.android.debug.armv7.neon.o):bands.c:function quant_partition: error: undefined reference to 'alg_quant'
drivers\libdrivers.android.debug.armv7.neon.a(bands.android.debug.armv7.neon.o):bands.c:function quant_partition: error: undefined reference to 'alg_unquant'
drivers\libdrivers.android.debug.armv7.neon.a(bands.android.debug.armv7.neon.o):bands.c:function quant_partition: error: undefined reference to 'renormalise_vector'
collect2.exe: error: ld returned 1 exit status

=====
scons: *** [bin\libgodot.android.debug.armv7.neon.so] Error 1
scons: building terminated because of errors
```

**Comment by akien-mga** [comment]
Could you post the full build log (console output) to a pastebin service (e.g. hastebin.com)?

**Comment by akien-mga** [comment]
Also, what platform are you building on and with what compiler?

**Comment by Abhicse079** [comment]
scons platform=android target=release_debug
http://hastebin.com/epokatohaf.tex


I am using windows 10 , 64 bit Machine and g++ compiler.

**Comment by akien-mga** [comment]
Thanks, the log shows that it's not only opus that doesn't get linked properly, but apparently also webp, vorbisfile, png, theora, etc., i.e. most of the libs of the monolithic driver.

**Comment by Abhicse079** [comment]
any solution.

**Comment by vnen** [comment]
I think this is the same as #3122.

@Abhicse079 try to reenable the libdrivers splitting by editing `drivers/SCsub` around line 66 and changing `if(False)` to `if(True)`.

**Comment by Abhicse079** [comment]
After  editing drivers/SCsub around line 66 and changing if(False) to if(True)

drivers\libdrivers1.android.opt.debug.armv7.neon.a(pngrutil.android.opt.debug.armv7.neon.o):pngrutil.c:function png_read_filter_row: error: undefined reference to 'png_init_filter_functions_neon'
collect2.exe: error: ld returned 1 exit status

=====
scons: *** [bin\libgodot.android.opt.debug.armv7.neon.so] Error 1
scons: building terminated because of errors.

**Comment by vnen** [comment]
This last error is caused by the split of libdrivers. We need a proper workaround against the command-line limit for Windows.

I still think it's the same error as #3122. Fixing one will fix both.

**Comment by reduz** [comment]
but i think there is code to fix the command line length limit in detect.py, i'm not sure what's going on myself

**Comment by reduz** [comment]
Can anyone running windows with all these things configured lend a hand on this?

**Comment by Abhicse079** [comment]
No any solution .. Moving to linux or any other game engine

**Comment by Abhicse079** [comment]
Used  <a href="https://github.com/godotengine/godot/tree/a028840ea5cfc6ac4e80d99c4850ae3f10eeb353">2418c</a>  working fine

**Comment by reduz** [comment]
@Abhicse079 you can always compile using visual studio

**Comment by reduz** [comment]
seems to work for me, combined a few options for this. Let me know how it goes

**Comment by Abhicse079** [comment]
It's frustrating implemented lot of suggestion still getting 

drivers\libdrivers1.android.opt.debug.armv7.neon.a(pngrutil.android.opt.debug.armv7.neon.o):pngrutil.c:function png_read_filter_row: error: undefined reference to 'png_init_filter_functions_neon'
collect2.exe: error: ld returned 1 exit status

scons: *** [bin\libgodot.android.opt.debug.armv7.neon.so] Error 1
scons: building terminated because of errors.


Do you have any solution.

**Comment by vnen** [comment]
It seems not to be fixed :disappointed:

**Comment by Abhicse079** [comment]
Do we have any commit that can be compile ?

**Comment by volzhs** [comment]
I don't know how much benefit I can get with png arm opt.
I just give up png arm opt and make compile successful on windows.
I changed like below at line 25 in ``drivers/png/SCsub``.

```python
import os
if ("neon_enabled" in env and env["neon_enabled"]) and os.name!="nt":
```

**Comment by akien-mga** [comment]
Fixed by #3975.
